tensorflow/tensorboard/components/vz-line-chart/vz-line-chart.ts

```diff
@@ -45,7 +45,7 @@ module VZ {
     private smoothingEnabled: Boolean;
 
     constructor(
-        xType: string, colorScale: Plottable.Scales.Color,
+        xType: string, yScale: string, colorScale: Plottable.Scales.Color,
```

Comment:
"yScale" as a variable name makes me expect a scale. Please rename to "yScaleType" or similar.